airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -506,9 +488,12 @@ def __init__(  # pylint: disable=too-many-arguments
                 del kwargs['params']
 
             # Create cluster object from kwargs
-            kwargs['region'] = region
-            kwargs['project_id'] = project_id
-            cluster = ClusterGenerator(**kwargs).make()
+            if project_id is None:
+                raise AirflowException(
+                    "project_id argument is required when building cluster from keywords parameters"
```

Comment:
That should not be problem when migrating from 1.10.X becaue:
https://github.com/apache/airflow/blob/827a717fecc675b4d58fa202be003dee8e423632/airflow/contrib/operators/dataproc_operator.py#L192-L193